packages/stripe_platform_interface/lib/src/method_channel_stripe.dart

```diff
@@ -206,6 +207,13 @@ class MethodChannelStripe extends StripePlatform {
         .parse(result: result!, successResultKey: 'token');
   }
 
+  @override
+  Future<void> dangerouslyUpdateCardDetails(CardDetails card) async {
```

Comment:
This function do not return some error natively in case we provide wrong details (e.g. non existing card number?  IF not we should also add to the docs that users of the method `dangerouslyUpdateCardDetails` should make sure they have their own validation.

Reply:
yes, no errors - I'll update the docs